rb/lib/htmlproofer/enforce_img_https.rb

```diff
@@ -0,0 +1,17 @@
+class EnforceImgHTTPS < ::HTMLProofer::Check
+  def run
+    @html.css('img').each do |node|
+      img = create_element(node)
```

Comment:
You might have misunderstood my intent in using the underlying `node`, which was to accomodate this:

`  <img src="//adresse.data.gouv.fr/static/img/BAN.png">`

which will now be considered incorrect by EnforceImgHTTPS, even though it is actually just fine. The problem is that the HTMLProofer::Element constructor provides "http" to make up for a missing `scheme`, and so our tests would fail in a case like the above.

**Right now** this doesn't lead to any failures, and so might not be an obstacle to merging as-is, but we might face a nasty surprise (and a long debugging session) later on if someone use `scheme`-less images (just as we now rely on `scheme`-less JS and CSS), and unexpectedly sees their PR go red.